docs/images/contributor-flowchart.svg

```diff
@@ -0,0 +1,182 @@
+<rect x="547.5" y="340.5" width="160" height="63" fill="#6929C4" stroke="#491D8B"/>
+<text fill="white" xml:space="preserve" style="white-space: pre" font-family="IBM Plex Sans" font-size="12" letter-spacing="0.32px"><tspan x="555" y="360.5">Do you know which Qiskit </tspan><tspan x="555" y="376.5">package you would like </tspan><tspan x="555" y="392.5">your feature added to?</tspan></text>
+<text fill="#121619" xml:space="preserve" style="white-space: pre" font-family="IBM Plex Sans" font-size="12" letter-spacing="0.32px"><tspan x="420" y="892.5">Yes</tspan></text>
+<text fill="#121619" xml:space="preserve" style="white-space: pre" font-family="IBM Plex Sans" font-size="12" letter-spacing="0.32px"><tspan x="511" y="892.5">No</tspan></text>
+<text fill="#121619" xml:space="preserve" style="white-space: pre" font-family="IBM Plex Sans" font-size="12" letter-spacing="0.32px"><tspan x="293" y="301.5">Yes</tspan></text>
+<text fill="#121619" xml:space="preserve" style="white-space: pre" font-family="IBM Plex Sans" font-size="12" letter-spacing="0.32px"><tspan x="236" y="300.5">No</tspan></text>
+<path d="M532 906V885.582C532 882.821 529.761 880.582 527 880.582H479C476.239 880.582 474 878.344 474 875.582V853" stroke="#343A3F"/>
+<rect x="520.5" y="744.5" width="101" height="95" fill="#6929C4" stroke="#491D8B"/>
+<rect x="630.5" y="744.5" width="106" height="95" fill="#6929C4" stroke="#491D8B"/>
+<rect x="570.5" y="625.5" width="115" height="47" fill="#6929C4" stroke="#491D8B"/>
+<text fill="#121619" xml:space="preserve" style="white-space: pre" font-family="IBM Plex Sans" font-size="12" letter-spacing="0.32px"><tspan x="528" y="701.5">work is in scope</tspan></text>
+<text fill="#121619" xml:space="preserve" style="white-space: pre" font-family="IBM Plex Sans" font-size="12" letter-spacing="0.32px"><tspan x="637" y="703.5">work is not in scope</tspan></text>
+<text fill="white" xml:space="preserve" style="white-space: pre" font-family="IBM Plex Sans" font-size="12" letter-spacing="0.32px"><tspan x="578" y="645.5">Discuss your idea </tspan><tspan x="578" y="661.5">with maintainers</tspan></text>
+<text fill="white" xml:space="preserve" style="white-space: pre" font-family="IBM Plex Sans" font-size="12" letter-spacing="0.32px"><tspan x="528" y="764.5">Work with </tspan><tspan x="528" y="780.5">maintainers to </tspan><tspan x="528" y="796.5">integrate your </tspan><tspan x="528" y="812.5">feature into </tspan><tspan x="528" y="828.5">Qiskit</tspan></text>
+<text fill="white" xml:space="preserve" style="white-space: pre" font-family="IBM Plex Sans" font-size="12" letter-spacing="0.32px"><tspan x="636" y="763.5">Publish the </tspan><tspan x="636" y="779.5">feature yourself </tspan><tspan x="636" y="795.5">and consider </tspan><tspan x="636" y="811.5">joining the </tspan></text>
```

Comment:
`Publish the feature yourself and consider joining the Qiskit Ecosystem` -> `Consider creating a stand-alone module and join the Qiskit Ecosystem!`

Reply:
✅ done